darts/models/forecasting/forecasting_model.py

```diff
@@ -1555,8 +1555,8 @@ def save(self, path: Optional[Union[str, BinaryIO]] = None, **pkl_kwargs) -> Non
         ----------
         path
             Path or file handle under which to save the model at its current state. If no path is specified, the model
-            is automatically saved under ``"{ModelClass}_{YYYY-mm-dd_HH:MM:SS}.pkl"``.
-            E.g., ``"RegressionModel_2020-01-01_12:00:00.pkl"``.
+            is automatically saved under ``"{ModelClass}_{YYYY-mm-dd_HH_MM_SS}.pkl"``.
```

Comment:
Not sure to understand what you refer to by "deep learning model docs", is it the docstring above each model constructor? If so, they do not make mention of the `save()` `path` argument.

However, I noticed that the docstring of the `model_name` argument was not consistent with the code, namely the described default string is "YYYY-mm-dd_HH:MM:_torch_model_run_PID" whereas the code was generating "YYYY-mm-dd_HH.MM.SS._torch_model_run_PID" (`.` instead of `:`, already compatible with Windows). I suppose that we should probably use the same convention for `model_name` and default `path` in `save()`: `_` or `.` to separate the hours, minutes and second?